<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 149, 148);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>671</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 223, 187);
 border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PUSH1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH8">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>390</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>390</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PUSH6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>390</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 20px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ganador_LB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>751</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PUSHRESET_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/Imagenes/escoba.png);
	font: 75 16pt &quot;Agency FB&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(242, 84, 45);
    border: 5px solid rgb(86, 44, 44);
    border-radius: 30px;
}
QPushButton:hover {
    background-color: rgb(18, 116, 117) ;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
